gradle/libs.versions.toml

```diff
@@ -23,8 +23,9 @@ maven = "3.8.7"
 mavenResolver = "1.7.3"
 micrometer = "1.10.4"
 nessieClientVersion = "0.44.0"
+# keep both opentelemetry versions below in sync
 opentelemetry = "1.23.1"
-opentelemetryAlpha = "1.20.1-alpha"
+opentelemetryAlpha = "1.23.1-alpha"
```

Comment:
wondering why renovatebot did not upgrade `opentelemetryAlpha` automatically :thinking: